### ISSUE TITLE
[Bug] FreeCAD Freezes/Locks when activating Fullscreen mode (View → Fullscreen)

### ISSUE THREAD

**Issue by Palmtree3** [FreeCAD/FreeCAD#7563]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Forums discussion

https://forum.freecadweb.org/viewtopic.php?f=3&t=72112

### Version

0.21 (Development)

### Full version info

```shell
OS: Windows 10 Version 2009
Word size of FreeCAD: 64-bit
Version: 0.21.30430 (Git)
Build type: Release
Branch: master
Hash: 9e1ed4a68ad43309c2b38f7f5efbb1feac4323cb
Python 3.10.6, Qt 5.15.4, Coin 4.0.0, Vtk 9.1.0, OCC 7.6.3
Locale: English/Australia (en_AU)
Installed mods: 
  * Behave-Dark-Colors 0.1.1
  * BIM 2021.12.0
  * Design456 0.0.1
  * dodo
  * Dracula 0.0.2
  * fasteners 0.4.10
  * Help 1.0.3
  * Parts_library
  * PieMenu
  * ProDarkThemePreferencePack 1.0.0
  * Reinforcement
  * SketchArch
```

### Issue description

Please refer to forum discussion ( https://forum.freecadweb.org/viewtopic.php?f=3&t=72112) 

### Anything else?

Images included in the forum post

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by kzootech** [comment]
I have made a pull request #8027 which I believe will fix this issue. It is caused by an limitation of Windows, OpenGL, and Qt. The pull request contains the recommended solution to this problem as detailed in [the Qt documentation](https://doc.qt.io/qt-5/windows-issues.html#fullscreen-opengl-based-windows). Putting this code in MainWindow::loadWindowSettings() seemed like the most appropriate place, any earlier in the window initialization and it does not seem to work, though a maintainer of FreeCAD may know better than me.

#8027

**Comment by donovaly** [comment]
Fixed by #8027
